chainer/function_hook.py

```diff
@@ -118,7 +136,8 @@ def deleted(self, function=None):
 
         Args:
             function(~chainer.FunctionNode): Function object to which
```

Comment:
(again, suggestion outside of this change)
to which -> from which

Reply:
Fixed.